generators/client-2/templates/src/main/webapp/app/account/_account.module.ts

```diff
@@ -18,7 +18,15 @@ import {
     PasswordComponent,
     PasswordResetInitComponent,
     PasswordResetFinishComponent,
-    SettingsComponent
+    SettingsComponent,
+<% if (enableSocialSignIn) { %>
+    JhSocialComponent,
```

Comment:
If a component/service/directive is used more than once in the app move those to the shared folder under a social sub folder